app/posts/how-to-use-cross-government-slack.md

```diff
@@ -0,0 +1,50 @@
+---
+title: How to use cross-government Slack
+description: Tips on how to join and make the most of the cross-government Slack workspace.
+date: 2023-06-27
+tags: post
+layout: post
+author:
+  name: Steve Messer
+  url: https://github.com/stevenjmesser
+image:
+  src: 
+  alt: 
+  opengraphImage: true
+---
+
+[Cross-government Slack](https://ukgovernmentdigital.slack.com/) is a Slack workspace for anyone working in central government, with channels on a range of topics. It's a place to connect with other people working on digital services across government.
+
+## A bit of history
+
+The cross-government Slack was created by the [Government Digital Service](https://www.gov.uk/government/organisations/government-digital-service) (GDS) in 2015 as a way for GDS to collaborate with other public sector digital service teams.
+
+Over time, the workspace naturally grew and became a place to share knowledge on all things about digital, data and technology in the public sector.
+
+The workspace is overseen by the [Central Digital & Data Office](https://www.gov.uk/government/organisations/central-digital-and-data-office).
+
+## Why should you use it?
+
+There's tons of reasons to use cross-government Slack. Here's a list of what I've found useful, but there's probably parts of the workspace I haven't made use of yet!
+
+- Network with other people in similar roles, for sharing techniques and tools
+- Find out about new programmes of work or job vacancies
+- Learn from other teams on how they approach accessibility
+- Discuss and share examples of applying the GOV.UK Design System to your service
+- Get help and support using GOV.UK Pay and GOV.UK Notify
+- Find meeting rooms in other government buildings
+- Keep up with what other people are working on in the [#weeknotes](https://ukgovernmentdigital.slack.com/archives/CE2K1LU3Y) channel
```

Comment:
(Me being overly finicky, part 2) Just to be consistent with Markdown used in other posts:

```suggestion
* Network with other people in similar roles, for sharing techniques and tools
* Find out about new programmes of work or job vacancies
* Learn from other teams on how they approach accessibility
* Discuss and share examples of applying the GOV.UK Design System to your service
* Get help and support using GOV.UK Pay and GOV.UK Notify
* Find meeting rooms in other government buildings
* Keep up with what other people are working on in the [#weeknotes](https://ukgovernmentdigital.slack.com/archives/CE2K1LU3Y) channel
```